docs/source/common/lightning_module.rst

```diff
@@ -731,13 +691,90 @@ a ``LightningModule``.
     with torch.no_grad():
         y_hat = model(x)
 
+
+-----------
+
+
+*************
+Child Modules
```

Comment:
I am not sure we want to advise this anymore as it is better to follow the design pattern to simplify using nn.Module without Lightning.

Reply:
I think it's okay to have as long as all are under `nn.Module`. It represents that LightningModule is just ``nn.Module`` and one can bring modules from outside.